source/_integrations/plugwise.markdown

```diff
@@ -236,33 +240,49 @@ script:
           preset_mode: asleep
 ```
 
-### Supported devices
+## Supported devices
+
+### Adam
+
+A complete zone control system also known as Adam HA, supporting:
+
+- On/Off, OpenTherm or Loria/Thermastage heating, and cooling support.
+- Running firmwares v3.x or v2.3.
+- Additional devices:
+  - Zone thermostats such as Lisa or Anna (see warning below on Anna),
+  - A temperature sensor, Jip,
+  - Valve controllers called Floor or Tom,
+  - An under-floor heating controller Koen (note: a Koen always comes with a Plug, which is the active part),
+  - Smart switches, either Plug or Aqara Smart Plug.
+
+{% note %}
+You can also use the Adam SA (Stand-alone, no Zigbee stick present) to make a single zone thermostat. 
+{% endnote %}
+
+
+{% endtip %}
+
+### Anna
```

Comment:
Can we provide a link here? https://www.plugwise.com/product/anna/?lang=en
Because below one-liner is far from complete.